src/app/shared/footer/footer.component.scss

```diff
@@ -0,0 +1,15 @@
+@use '@angular/material' as mat;
+@import '../../styles/palettes';
```

Comment:
Trzeba to zmienić na @use, Grzesiek na discordzie napisał w jaki sposób to zrobić